tests/testthat/test-population_scaling.R

```diff
@@ -208,9 +207,9 @@ test_that("test joining by default columns", {
     recipes::step_naomit(recipes::all_predictors()) %>%
     recipes::step_naomit(recipes::all_outcomes(), skip = TRUE)
 
-  suppressMessages(prep <- prep(r, jhu))
+  expect_silent(prep(r, jhu))
 
-  suppressMessages(b <- bake(prep, jhu))
+  expect_silent(bake(prep(r, jhu), new_data = NULL))
```

Comment:
question: is the test here really about expecting silence or is it just a one-time check that the suppressMessages is no longer needed? if the latter, then we probably can just call these normally? though I'm not sure if these even do anything, since they don't reassign the recipe?